docs/walkthrough-1.7.md

```diff
@@ -3,6 +3,323 @@
 This document assumes that you've installed Service Catalog onto your cluster.
 If you haven't, please see [install-1.7.md](./install-1.7.md).
 
-This document is a work in progress. Instructions for the self-guided demo
-will be similar to those in the [1.6 walkthrough](./walkthrough-1.6.md),
-but note that the commands in that document will not work as-is.
+All commands in this document assume that you're operating out of the root
+of this repository.
+
+# Step 1 - Installing the UPS Broker Server
+
+In order to effectively demonstrate the service catalog, we need a sample
+broker server. We've built a "dummy" server called the User Provided Service
+(UPS) broker. You can find the codebase for that broker
+[here](https://github.com/kubernetes-incubator/service-catalog/tree/master/contrib/pkg/broker/user_provided/controller).
+
+We're going to deploy that broker to our Kubernetes cluster before proceeding,
+and we'll do so with the UPS helm chart. You can find details about that chart
+in the chart's 
+[README](https://github.com/kubernetes-incubator/service-catalog/blob/master/charts/ups-broker/README.md).
+
+Otherwise, to install with sensible defaults, run the following command:
+
+```console
+helm install charts/ups-broker --name ups-broker --namespace ups-broker
+```
+
+# Step 2 - Creating a `ServiceBroker` Resource
+
+Because we haven't created any resources in the service-catalog API server yet,
+`kubectl get` will return an empty list of resources.
+
+```console
+kubectl get brokers,serviceclasses,instances,bindings
+No resources found
+```
+
+We'll register a broker server with the catalog by creating a new
+[`ServiceBroker`](../contrib/examples/walkthrough/ups-broker.yaml) resource.
+Do so with the following command:
+
+```console
+kubectl --context=service-catalog create -f contrib/examples/walkthrough/ups-broker.yaml
```

Comment:
We don't need the context anymore here I think.

Reply:
good catch - thanks @kibbles-n-bytes

Reply:
done